xdsl/transforms/csl_stencil_bufferize.py

```diff
@@ -370,9 +375,11 @@ def match_and_rewrite(self, op: arith.ConstantOp, rewriter: PatternRewriter, /):
             return
         assert isinstance(op.value, DenseIntOrFPElementsAttr)
         assert isa(op.value.type, TensorType[Attribute])
-        typ = DenseIntOrFPElementsAttr(
-            [tensor_to_memref_type(op.value.type), op.value.data]
+        memref_type = cast(
+            RankedStructure[AnyFloat | IntegerType | IndexType],
+            tensor_to_memref_type(op.value.type),
         )
+        typ = DenseIntOrFPElementsAttr.from_list(memref_type, op.value.get_values())
```

Comment:
why this change? it should still work, as-is, no?

Reply:
yes, but then you have to access the private _data.
I can unprivatize data and make exceptions for these more trivial cases